11-payment-encoding.md

```diff
@@ -285,7 +302,7 @@ Breakdown:
   * `xysxxatsyp3k7enxv4js`: '1 cup coffee'
 * `x`: expiry time
   * `qz`: `data_length` (`q` = 0, `z` = 2. 0 * 32 + 2 == 2)
-  * `pu`: 60 seconds (`p` = 1, `u` = 28.  1 * 32 + 28 == 60)
+  * `pu`: 60 seconds (`p` = 1, `u` = 28. 1 * 32 + 28 == 60)
```

Comment:
...28; 1... use semicolon or colon instead of period?

Reply:
think you're right; that's more readable. Changing throughout.